crates/matrix-sdk-crypto/src/identities/device.rs

```diff
@@ -159,10 +159,82 @@ impl Device {
     /// can be confirmed as the creator and owner of the `m.room_key`.
     pub fn is_owner_of_session(&self, session: &InboundGroupSession) -> bool {
         if session.has_been_imported() {
+            // An imported room key means that we did not receive the room key as a
+            // `m.room_key` event when the room key was initially exchanged.
+            //
+            // This could mean a couple of things:
+            //      1. We received the room key as a `m.forwarded_room_key`
+            //      2. We imported the room key through a file export.
+            //      3. We imported the room key through a backup.
```

Comment:
```suggestion
            //      1. We received the room key as a `m.forwarded_room_key`,
            //      2. We imported the room key through a file export,
            //      3. We imported the room key through a backup.
```

Reply:
not that it matters a great deal, but just fyi we use a different convention in matrix spec docs: https://github.com/matrix-org/matrix-spec/blob/main/meta/documentation_style.rst#general

Reply:
Added the missing periods https://github.com/matrix-org/matrix-rust-sdk/pull/1371/commits/4e5c9c4a4881670a70fc6592717dd37d16538e60.